### PR TITLE
July Refresh

### PR DESCRIPTION
**Pull request by tehcrashxor**
[AB#3536688](https://dev.azure.com/dynamicscrm/1fb98997-2f9e-4734-be8a-9728680447c2/_workitems/edit/3536688)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 [Power Platform CLI Exposed](https://www.youtube.com/playlist?list=PLlrxD0HtieHhEdLHxQOU96ySSZpMCyAxf)
 
 ## Release Notes
+REPLACE_ME_WITH_VERSION:
+  - pac CLI REPLACE_ME_WITH_VERSION (July refresh, see release notes on [nuget.org](https://www.nuget.org/packages/Microsoft.PowerApps.CLI/))
+  - Authentication Settings - Can now choose other Azure clouds for the "New Auth Profile" button to target [#619](https://github.com/microsoft/powerplatform-vscode/pull/619)
+  - Environments & Solutions panel improvements:
+    - Environments now display an icon for the active environment, and button added to select a different environment [#629](https://github.com/microsoft/powerplatform-vscode/pull/629)
+    - Solutions now have tooltip and icon to indicated managed vs. unmanaged solutions [#618](https://github.com/microsoft/powerplatform-vscode/pull/618)
```
Comment (tyaginidhi) [inline README.md:27]: - Conflict resolution optimization for incoming changes accept scenario [#664](https://github.com/microsoft/powerplatform-vscode/pull/664)
- Optimized load of webfiles for old data model - media files are now loaded only on view a specific file action [#662](https://github.com/microsoft/powerplatform-vscode/pull/662)
- Performance optimizations and better support for revisit sessions file load [#646](https://github.com/microsoft/powerplatform-vscode/pull/646)

**Review by tyaginidhi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tehcrashxor) [reply README.md:27]: added

**Review by petrochuk — APPROVED** [review]

**Review by petrochuk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -19,6 +19,17 @@ Installing this extension will also make the latest Power Platform CLI (aka pac)
 [Power Platform CLI Exposed](https://www.youtube.com/playlist?list=PLlrxD0HtieHhEdLHxQOU96ySSZpMCyAxf)
 
 ## Release Notes
+2.0.1:
```
Comment (amitjoshi438) [inline README.md:22]: - Add AI-generated code using Power Pages Copilot (preview)

**Review by petrochuk — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (tyaginidhi) [reply README.md:22]: For this release, note that this capability is just available for vscode desktop and not web extension

**Pull request merged**